spec/requests/passwords_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+RSpec.describe 'Passwords', type: :request do
+  let(:user) { create :user }
+
+  describe 'POST /create' do
+    it 'resets password' do
+      post '/api/auth/password', params: {
+        email: user.email,
+        redirect_url: 'http://localhost:3000/reset-password'
+      }
```

Comment:
Put this post request outside of the it block and set it to run before each test case